src/plugins/intel_cpu/tests/unit/shape_inference_test/prior_box_clustered_shape_inference_test.cpp

```diff
@@ -117,6 +131,12 @@ TEST_F(PriorBoxClusteredV0StaticShapeInferenceTest, invalid_number_of_elements_i
                                     {{0, std::make_shared<HostTensor>(element::i64, ov::Shape{3}, output_size)}}),
                     NodeValidationFailure,
                     HasSubstr("Output size must have two elements"));
+    // TODO , implementation should throw execption
```

Comment:
If this cpu unit  would be part dedicated test fixture for cpu specific shape inference test then test code can be implemented and the GTEST_SKIP() macro can be used to mark it as not implemented yet.

Reply:
fixed